docs/ref/models/querysets.txt

```diff
@@ -1863,16 +1863,19 @@ Returns a tuple of ``(object, created)``, where ``object`` is the retrieved or
 created object and ``created`` is a boolean specifying whether a new object was
 created.
 
-This is meant as a shortcut to boilerplatish code. For example::
+This is meant as a shortcut to boilerplatish code and to prevent duplicate objects
+from being created when requests are made in parallel. For example::
 
     try:
         obj = Person.objects.get(first_name='John', last_name='Lennon')
     except Person.DoesNotExist:
         obj = Person(first_name='John', last_name='Lennon', birthday=date(1940, 10, 9))
         obj.save()
 
-This pattern gets quite unwieldy as the number of fields in a model goes up.
-The above example can be rewritten using ``get_or_create()`` like so::
+This pattern gets quite unwieldy as the number of fields in a model goes up,
+and if multiple requests are made concurrently, two ``Person`` objects with the
+same name will be created. The above example can be rewritten using
```

Comment:
Replace "will be created" with "can be created" as it depends on the threading, the model's uniqueness constraints, and whether surrounding code triggers a lock of some other kind.

Reply:
Good point. I went with "may be created" since it sounds slightly more natural to me than "can be created".